plugin/manager.go

```diff
@@ -173,9 +173,22 @@ func (pm *Manager) reload() error { // todo: restore
 		if validFullID.MatchString(v.Name()) {
 			p, err := pm.loadPlugin(v.Name())
 			if err != nil {
-				return err
+				if os.IsNotExist(errors.Cause(err)) {
+					// Likely some error while removing on an older version of docker
+					logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Warn("Missing plugin config, skipping. This may be caused due to a failed remove and requires manual cleanup.")
+				} else {
+					logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Error("Error loading plugin, skipping.")
```

Comment:
What's the difference between 1 plugin failing vs all plugins failing if there's only 1 plugin loaded?

The systems that require failed plugins will error out anyway. I don't think it makes sense to fail the daemon because a plugin couldn't load.